.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,24 @@
+name: Generate Preview
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  deploy_preview:
+    name: Deploy Preview of PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - run: npm install yaml @actions/github
+      - run: echo "::set-env name=worker::$( node ./scripts/verify.js ${{ github.event.comment.user.login }} "${{ github.event.comment.body }}" )"
+      - if: env.worker == 'true'
```

Comment:
I also considered performance of job. So job will further execute only if ```verified``` ***user*** and ***command***.